esy-lib/Tarball.re

```diff
@@ -87,6 +87,19 @@ let unpackWithTar = (~stripComponents=?, ~dst, filename) => {
   | Some(stripComponents) =>
     Fs.withTempDir(out => {
       let* () = unpack(out);
+      let normalizedOut = EsyBash.normalizePathForCygwin(Path.show(out));
+      let normalizedOut =
+        Fpath.of_string(normalizedOut) |> Stdlib.Result.get_ok;
```

Comment:
Suggestion

```reason
      let normalizedOut = out |> Path.show |> EsyBash.normalizePathForCygwin |> Path.ofString
```

Let's use `errorf` from `EsyLib` should anything fail, instead of just unwrapping with `Result.get_ok`